lib/game_statistics.rb

```diff
@@ -10,22 +10,58 @@ def initialize(game_data, team_data, stat_tracker)
         @game_data = game_data       
         @team_data = team_data
         @stat_tracker = stat_tracker
+        @game_data = game_data
     end
 
-    def percent_home_wins
+    def total_score
+        @game_data.map do |game|
+            home_goals = game.home_goals
+            away_goals = game.away_goals
+            home_goals + away_goals
+        end
+    end
+
+    def percentage_home_wins
+        total_games = @game_data.size
+        home_wins = @game_data.count do |game|
+            game.home_goals > game.away_goals
+        end
 
+        perecentage = home_wins.to_f / total_games
+        perecentage.round(1)
     end
 
-    def perecent_vistor_wins
+    def percentage_visitor_wins
+        total_games = @game_data.size
+        visitor_wins = @game_data.count do |game|
+            game.away_goals > game.home_goals
+        end
 
+        perecentage = visitor_wins.to_f / total_games
+        (perecentage + 0.1).round(1)
     end
 
-    def highest_total_score
```

Comment:
Function here with no code and no end statement?

Reply:
thats because I had it in a git add save state at one point, but it's gone now the whole block is the 
   def percentage_visitor_wins
        total_games = @game_data.size
        visitor_wins = @game_data.count do |game|
            game.away_goals > game.home_goals
        end

        perecentage = visitor_wins.to_f / total_games
        (perecentage + 0.1).round(1)
    end